scripts/tasks/lint-imports.js

```diff
@@ -154,19 +169,43 @@ function lintImports() {
         const parts = importStatementRegex.exec(statement);
 
         if (parts) {
-          _evaluateImport(filePath, parts[1], importErrors, isExample);
+          _evaluateImport(filePath, parts, importErrors, isExample);
         }
       });
     }
+
+    if (isExample) {
```

Comment:
Starting to think that we are slowly building our own eslint here (complete with plugins!) Overall, maybe we can reimplement this as a tslint / eslint plugin later not in your path to do editor work.

Reply:
Agreed!! But I definitely don't want to move it over now in the interest of actually getting the overall project done.